### PR TITLE
update v0.9.0

### PR DESCRIPTION
**Pull request by agourlay**
PR updating the current version to `v0.9.0`.

### CODE REVIEW

**Comment by agourlay** [comment]
https://github.com/qdrant/rust-client/pull/5

File: docs/redoc/v0.9.0/openapi.json [hunk 1]
```diff
@@ -0,0 +1,5093 @@
...
+    "contact": {
+      "email": "andrey@vasnetsov.com"
+    },
+    "license": {
+      "name": "Apache 2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    },
+    "version": "master"
```
Comment (generall) [inline docs/redoc/v0.9.0/openapi.json:2286]:
```suggestion
    "version": "v0.9.0"
```

Comment (agourlay) [reply docs/redoc/v0.9.0/openapi.json:2286]: Good catch, I completely missed that one :+1: 

I will rebase first to get your latest change on the transfer abort API.

**Comment by e-ivkov** [comment]
> ## drivers:
> - https://github.com/qdrant/rust-client/pull/5
> - https://github.com/qdrant/go-client/pull/8
> 
> ## docs:
> - https://github.com/qdrant/docs/pull/17

What about python client?

**Review by generall — APPROVED** [review]

**Pull request merged**